prompting/cleaners/cleaner.py

```diff
@@ -0,0 +1,53 @@
+from typing import List, Dict
+
+import bittensor as bt
+
+from prompting.cleaners.all_cleaners import RemoveQuotes, RemoveRoles, PruneEnding
+
+SUPPORTED_CLEANERS = {
+    "remove_quotes": RemoveQuotes,
+    "remove_roles": RemoveRoles,
+    "prune_ending": PruneEnding,
+}
+
+
+class CleanerPipeline:
```

Comment:
I kept this as a class because I instantiate a cleaner in Task.generate and I pass this directly to the HuggingFaceLLM.